sdk/go/common/util/cmdutil/term_test.go

```diff
@@ -393,7 +393,7 @@ func (p testProgram) Build(t *testing.T) (cmd *exec.Cmd) {
 		return exec.Command(nodeBin, append([]string{src}, p.args...)...)
 
 	case pythonTestProgram:
-		pythonBin := lookPathOrSkip(t, "python")
+		pythonBin := lookPathOrSkip(t, "python3")
```

Comment:
I'm curious if this "just works" on Windows, or if we need to make this smarter like this:

https://github.com/pulumi/pulumi/blob/4a12902bde0dbcb15351284f9d59f028ed271829/pkg/testing/integration/program.go#L931-L934

Reply:
Oh I think you are right https://github.com/python/cpython/issues/99185